calculus/integration-substitution.md

```diff
@@ -1 +1,45 @@
 # Integration using substitution
+## Skill Builder
+### Find General Solutions for:
+(a) $y' = (x+2)ln(x)$
+
+  $y = \int (x+2)\ln(x) \ dx$  
+    
+  $u = ln(x)$ $dv = (x+2)dx$  
+  $du = \frac{1}{x}dx$  $v = \frac{x^2}{2} + 2x$
+    
+ $=(\frac{x^2}{2} + 2x)ln(x)- \int (\frac{x^2}{2} + 2x) \frac{1}{x}\ dx$    
+ <br> $\to (\frac{x^2}{2} + 2x)ln(x)-\frac{x^2}{4}-2x+C$  
+<br>
+<br>
+
+(b) $y' = x^5sin(x)$<br>
+
+  $y = \int x^5sin(x) \ dx$
+
+  $u: \ x^5 \ 5x^4 \ 20x^3 \ 60x^2 \ 120x \ 120 \ 0$ <br>
+  $dv: \ sin(x) \\ -cos(x) \\ -sin(x) \\ cos(x) \\ sin(x) \\ -cos(x) \\ -sin(x)$  
```

Comment:
You're going to want commas to separate all of these things. Spaces are not enough. You should also example what is happening here as otherwise it will not make sense.